tests/test_type_caster_bare_interface.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+
+from pybind11_tests import type_caster_bare_interface as m
+
+
+def test_cast():
+    assert m.rtrn_valu() == "cast_rref"
+    assert m.rtrn_rref() == "cast_rref"
+    assert m.rtrn_cref() == "cast_cref"
+    assert m.rtrn_mref() == "cast_mref"
+    assert m.rtrn_cptr() == "cast_cptr"
+    assert m.rtrn_mptr() == "cast_mptr"
+
+
+def test_load():
+    assert m.pass_valu(None) == "load_valu"
+    assert m.pass_rref(None) == "load_rref"
+    assert m.pass_cref(None) == "load_cref"
+    assert m.pass_mref(None) == "load_mref"
+    assert m.pass_cptr(None) == "load_cptr"
+    assert m.pass_mptr(None) == "load_mptr"
```

Comment:
What is actually tested here? This is just about calling different functions and comparing their return values, isn't it?